### ISSUE TITLE
Otolith request review

### ISSUE THREAD

**Issue by bethanyriggle** [afsc-gap-products/Survey-Planning#31]

**Comment by zoyafuso-NOAA** [comment]
From Nancy: G:\SURVEY COLLECTIONS holds the otolith review requests from previous years.

**Comment by zoyafuso-NOAA** [comment]
@Ned-Laman-NOAA 

I used a bootstrapping technique to gauge how well the requested otolith sampling plan for 2022 would have met the requested otolith targets for the past five sampling years (2010, 2012, 2014, 2016, 2018). For each sampling year, I take a empirical bootstrap sample (sampling with replacement) of 420 hauls and apply each otolith collection rule [as requested for 2022](https://docs.google.com/spreadsheets/d/1W8bKigNipl5IdQIQH5Kyq4twJxZKLi1zeS9UNE_JAsU/edit#gid=1924553237).  I assumed that both vessels have the same collection rules although in the past, certain species had only one vessel collecting otoliths. The pdf shown [here](https://github.com/zoyafuso-NOAA/AIGOA_Core_Otolith/blob/main/results/species_bootstrap.pdf) shows the distribution of those bootstrapped collections. 

Some notes:
* Some species don't have a target total number collected (i.e., Atka mackerel, harlequin rockfish, shortspine thornyhead)
* Many of the requests will hit the target number of otoliths
* Some requests will not meet the target, i.e., Greenland turbot, rougheye rockfish, shortraker rockfish. Northern rock sole seems to be ever-present in the survey, is expected to exceed the 300 target by ~200
* Total otoliths collected per haul range are in the upper 10s/lower 20s. 

Should I redo the bootstrap with increased collections for those deficit species and see how that affects the total otolith per haul? What is the maximum otolith total load that we are okay with? I can update this comment with the observed distribution of total otoliths per haul for each year.

**Comment by Ned-Laman-NOAA** [comment]
This looks great, Zack! Answers to some of your questions.
For Atka, contact Sandra Lowe and ask her if the target of 1000 from 2020 still holds. It looks like her 2022 request would hit that mark closely enough.
For Greenland turbot, it looks like increasing the collection rubric above 15 wouldn't result in more specimens. Is that your read? If so, then we let Meaghan know that we appear to be collecting all of the turbos that there are available to the survey.
Ditto for rougheye and shortraker.
I think we need clarification from Carey about her NRS collection. I get collect none when < 1! How many do we collect when > 1? What did you assume when you projected the 2022 NRS collection?
For Jane's fishes, let's communicate what our projections produce and ask her if those targets are realistic to what can be aged or if she wants to adjust them.
Our per haul otolith capacity is in the high 20s so it sounds like we have a little room to add if needed.
How did that project total otolith collection for 2022 compare with the total number of otoliths collected in the last few AI surveys? Don't we typically come home with around 6000 otolith pairs?
About re-doing the bootstrap for the deficit species. That sounds like a good approach, although I suspect for some of those species we haven't been under collecting otoliths relative to how many we catch. 
As I indicated above, if we keep the average total number of otoliths collected between 25 and 28 (definitely below 30) we should be able to meet our collections and make our stations.

**Comment by zoyafuso-NOAA** [comment]
@Ned-Laman-NOAA 

I will contact SA authors tomorrow and cc you with those messages. I'll work on those bootstraps with increased collection rules for the deficit species and I'll note the capacity for total haul-level collections at 25-28. For NRS, I assumed if >= 2, collect 2, otherwise 0. I think Carey inputted 2 otoliths previously and maybe removed it (or I assumed it was 2 for the analysis) but I will ask about this.

This is the total otoliths collected over the years:

Year | Total Otoliths
-- | --
2010 | 7098
2012 | 6014
2014 | 6275
2016 | 6311
2018 | 6772

I'll add the same calculation for the total cruise-level collections.

This table shows the distribution of haul-level total otoliths collected observed in the past five survey years. It must have been a long day when they extracted 108 pairs of otoliths...

2010 |   |   |   |   |   |  
-- | -- | -- | -- | -- | -- | --
min | 2.50% | 25% | median | 75% | 97.50% | max
1 | 1 | 6 | 15 | 26 | 56 | 108
  |   |   |   |   |   |  
2012 |   |   |   |   |   |  
min | 2.50% | 25% | median | 75% | 97.50% | max
1 | 1 | 5 | 11 | 22 | 55 | 85
  |   |   |   |   |   |  
2014 |   |   |   |   |   |  
min | 2.50% | 25% | median | 75% | 97.50% | max
1 | 1 | 6 | 14 | 24 | 50.15 | 70
  |   |   |   |   |   |  
2016 |   |   |   |   |   |  
min | 2.50% | 25% | median | 75% | 97.50% | max
1 | 3 | 9 | 13 | 20 | 40 | 77
  |   |   |   |   |   |  
2018 |   |   |   |   |   |  
min | 2.50% | 25% | median | 75% | 97.50% | max
1 | 4 | 12 | 17 | 22 | 31 | 36

**Comment by zoyafuso-NOAA** [comment]
Updates from last week, refer to the [output document](https://github.com/zoyafuso-NOAA/AIGOA_Core_Otolith/blob/main/results/species_bootstrap.pdf) for the following notes. These plots show the distribution of bootstrapped total otolith collected under the requested/modified 2022 collection rules with the targets in dashed lines.

1)  Sandra Lowe confirmed a target of 1000 for Atka mackerel.
2)  I have contacted Meaghan Bryan, Paul Spencer, and Jane Sullivan about the targets for Greenland turbot rougheye rockfish, and harlequin rockfish will probably not be fulfilled. 
3) Carey McGilliard requested a collected rule that would lead to a target similar to what was historically collected, so I bumped the target to 500 for NRS and 300 for SRS. For SRS, that corresponds to 12 SRS collected in the SBS and EAI. For NRS, that corresponds to 2 NRS when >=2. 
4) Shortraker rockfish is probably the only deficit species we can increase the collection rule to get a higher target, so I can see us shifting it up to 12 or 15 (from 8). Given that shortrakers are often quite large, is there a rule of thumb on board as to the maximum we would process in one haul? In the output, I assumed 12 shortakers collected. 
5) I communicated the different target levels for Jane Sullivan's fishes. Collecting up to 4 SST and up to 10 duskies will produce target ranges that match historical levels (~350-400 for SST and 100 for duskies). 
6) I added another boxplot time series for total otoliths collected and the proportion of hauls with > 30 otoliths (something we want to minimize).  Historically, the total otoliths collected was between 6000-7000. With these projection rules, the range is about 7500-8000. This seems high but the target total otoliths is 7700 (if you were to add the targets of all species) so we should be expecting a total otolith count across species to be between 7000-8000. 
7) Bootstrapped haul-level total otoliths were between 15-20 (median). 4-11% of the bootstrapped hauls collected > 30 otoliths.

**Comment by Ned-Laman-NOAA** [comment]
Point-by-point response
1 - thanks for confirming
2 - How did the SAs respond to our likely inability to meet their target specimen quantities?
3 - understood about adjusting SRS and NRS collections upward
4 - there is no rule of thumb about a max for shortrakers. we rarely get large numbers in single catches, but when we do we should take them
5 - Jane was OK with upping the SST and dusky collections? I am always curious to know if they align their targets with their ageing capacity, or at least think about it ahead of time
6 - I'm a little concerned that in recent Aleutian years we've collected closer to 6000 otoliths but are now targeting 7500-8000. Is that the sum of targets or the sum of projected collections given the sampling schemes? Ideally, the latter should land below 7000 based on recent survey yrs. Do you agree?
7 - based on the this outcome, I still think we should be looking for ways to keep the total project collection under 7000 otolith pairs.

**Comment by zoyafuso-NOAA** [comment]
2 - Meaghan, Paul, and Jane were fine with those species not reaching their targets. 
4 - Shortrakers are quite over-dispersed so playing around with the collection rules doesn't really change the totals collected by much unless you really bump it to something like 15. 
5 - From my conversations with Jane and Carey, it seems like they base their targets on what was previously collected. This is the reason why I proposed increasing the target for NRS, we been consistently collecting >> 300 in the past. My guess is that the extent to which these targets are determined by basing what was historically collected versus some sort of power analysis to determine a desirable target is SA-dependent. 
6 - The 7500-8000 target comes from adding all the requested targets for this year (7700 to be exact). It is higher than what was requested for 2018 because of increases in requested targets for PCod (from 500 to 700, requested by Ingrid), NRS (300 to 500, adjusted by me), northern RF (600 to 700, requested by Paul), harlequin RF (from 0 to 50, estimated by me), dusky RF (from 0 to 100, estimated by me). The total requests from 2018 was 7050, so yes, the target is likely higher than recent years. 
7 -- We could set the targets for PCod, NRF, and NRS back to what they were in 2018. The shortraker target could also be reduced to something like 200 or 250 because we have some confidence in saying that 350 is too high of a target. Same with Greenland turbot, a target like 100 (50 even) is slightly more realistic. That would get the total request target to be closer to 7000.

For the collection rules, reverting back to the 2018 rules for PCod or modifying it with a threshold (if < x don't collect) could help reduce the actual total otoliths collected but I don't know whether PCod (and pollock) are species we can be messing with decreasing collections without some political pushback. You could also look at reducing the collection rule for Atka mackerel, I noticed collections spiked in 2018 when the collection rules were changed. Is there a story to this, does it need to be so high?

**Comment by Ned-Laman-NOAA** [comment]
Thanks, Zack.
From what you've explained and put together, I think we are going to have to run with this total ask of 7700. I suspect that after a year without otoliths from the AI the collection need has increased slightly. If we hold true to form from what we saw in 2018 we collected around 5% less than the target asked for (6700 out of 7050) so perhaps we expect to collect something like 7400 otoliths total this summer. Still seems high, but doable.
Once you are satisfied that all of the negotiations are over and done (and I'm giving my approval here for what you have on paper), please communicate with Nancy Roberson that the otolith collections plans are final. 
Can you also let Nate know that we are targeting 7700 otolith specimens so that he can guesstimate the number of otolith boxes to load on our boats?
Great job dialing in these collections!

**Comment by nwraring** [comment]
Hi Ned and Zack. In the past Wayne has pretty much just told me how many
boxes to ask for. I'm thinning bout this and wondering if we have estimates
broken down by sp. Clearly if we were collecting 7700 from one sp. we'd
just divide that by 140 and add a couple of boxes and have our estimate. If
I can get an estimate per sp. I can estimate the number of boxes per sp as
well as the number of oversized I'll be needing to ask for.
Thanks,
nate

**Comment by Ned-Laman-NOAA** [comment]
@nwraring 
Once Zack has updated the AI CORE otolith requests [here](https://docs.google.com/spreadsheets/d/1W8bKigNipl5IdQIQH5Kyq4twJxZKLi1zeS9UNE_JAsU/edit?usp=sharing), you should be able to review the final version to answer your question regarding how many otolith boxes we're likely going to need.
BTW - who is cleaning and stuffing otolith boxes for us this year? I had heard that Dennis wasn't coming out this season.

**Comment by zoyafuso-NOAA** [comment]
Hi Ned, 

I have communicated with the assessors and we have agreed to the collection rules and any adjustments that I suggested. I will let Nancy know that the rules are finals. 

@nwraring: The [CORE spreadsheet](https://docs.google.com/spreadsheets/d/1W8bKigNipl5IdQIQH5Kyq4twJxZKLi1zeS9UNE_JAsU/edit#gid=1924553237) has the totals requested by species. If we end up collecting more otoliths than the target, is the rule to continue to collect otoliths so that we fully sample the sampling area? And 140 is the total number of vials for a standard collection box, right?
